filters/filters.go

```diff
@@ -117,6 +117,17 @@ type FilterContext interface {
 
 	// Performs a new route lookup and executes the matched route if any
 	Loopback()
+
+	Logger() FilterContextLogger
+}
+
+// FilterContextLogger is the logger object which can log messages from filter
+// including context information from this filter.
```

Comment:
`// FilterContextLogger is the logger which logs messages with additional context information`